### ISSUE TITLE
[BUG] App Screenshots do not appear

### ISSUE THREAD

**Issue by cjerrington** [Electron-Store/electron-app-store#19]
I really like this idea as I've used the electronjs.com/apps and have found a lot of exciting projects! It has the linux or app store kind of feel to the electron community. However, there are no images in the 0.0.4 release on Windows. The app icons are broken. 

**To Reproduce**
Windows 10
0.0.4 Release

**Expected behavior**
Show screenshots and app icons where appropriate

**Screenshots**
![electron-store](https://user-images.githubusercontent.com/8809376/163914241-c10a1988-edeb-4598-875b-49c9ae331fa5.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Build .exe
 - Version 0.0.4

**Additional context**
This might be due to the way you are reading the apps .yml data? I've noticed even on the electronjs.org/apps site, the screenshots do not appear there either.

**Comment by isworking** [comment]
Unfortunately, the v0.0.4 and older versions of the App Store use the older API which just parses the https://electronjs.org/apps website and the reason the screenshots and icons are not appearing is CORS Issue. I've created a new API that fixes the CORS Issue and many more issues but also introduces some major API changes. Therefore, the existing version of the App Store isn't compatible with it. So, I'm currently working on the full rewrite of the App Store. Currently, I'm the only contributor of this App Store so it's taking some more time.

**Comment by cjerrington** [comment]
Awesome! I saw that a 0.0.5 was mentioned in another issue so this looks to be resolved in upcoming releases? Should we keep this open until that release if others are experiencing the same thing? 

Sounds like you have a plan for the rewrite. I can try to help once that is completed!

**Comment by isworking** [comment]
You're right, we should keep this issue open until v0.0.5 is released. Also, most API-related issues are fixed with the new API which is incompatible with the current Electron App Store because it introduces some breaking changes. So, I'm currently working on the full rewrite of the App Store with React (+ Chakra UI) and TypeScript. The newer version will also have new design changes and a complete design revamp as well as many new and revamped features. But as I'm currently the only active maintainer of this project, it's taking more time than usual.
